airbyte-integrations/connectors/source-recurly/source_recurly/streams.py

```diff
@@ -0,0 +1,219 @@
+#
+# Copyright (c) 2021 Airbyte, Inc., all rights reserved.
+#
```

Comment:
Is it? You've run their license by legal?

Reply:
This is autogenerated when I ran the `gradle` build tool that runs some acceptance, integration and unit tests as well as code styling 😕